packages/react-native/src/attributes/NativeAttributeProvider.ts

```diff
@@ -0,0 +1,19 @@
+import { type AttributeType, type BacktraceAttributeProvider } from '@backtrace-labs/sdk-core';
+import { NativeModules } from 'react-native';
+export class NativeAttributeProvider implements BacktraceAttributeProvider {
+    public get type(): 'scoped' | 'dynamic' {
+        return this._type;
+    }
```

Comment:
You can rename `_type` in constructor to `type` and change it's accessor to `public`. This getter will be redundant then.

Reply:
Good idea. Changed